faq/index.md

```diff
@@ -52,6 +52,14 @@ and capabilities and use the errors response to let users know. This error featu
 is still pending to be included in the standard since is still in
 [discussion](https://github.com/json-api/json-api/issues/7).
 
+### Where's PUT? Can I use method *X* to do *Y*? <a href="#wheres-put" id="wheres-put" class="headerlink"></a>
+
+JSON API does not currently specify the use of the `PUT` method for any purpose.
+
+Servers may complement the base specification by providing extra capabilities and
+alternative ways of requesting certain operations (e.g., resource creation via
+`PUT` in addition to `POST`).
```

Comment:
The reason is that the HTTP spec reserves PUT for complete resource replacements, while PATCH is intended for partial updates. If the payload only specifies a strict subset of fields, a true PUT request would cause the omitted fields to be reset. PATCH causes their values to remain intact.

This PR was not originally intended to answer that question, hence #488. But if people think it would make sense to address the PUT/PATCH issue directly in this FAQ entry instead of adding a new one, then I'm certainly ready to add that explanation here.